playground/package.json

```diff
@@ -65,12 +65,11 @@
     "@commercetools-frontend/mc-scripts": "22.14.1",
     "@commercetools-test-data/commons": "4.11.1",
     "@formatjs/cli": "6.1.1",
-    "@types/dotenv-flow": "^3.3.1",
```

Comment:
Remove this dependency as it is mentioned in the `v4` [release notes](https://github.com/kerimdzhanov/dotenv-flow/releases/tag/v4.0.0).